doc/running.rst

```diff
@@ -4,6 +4,30 @@ Running MirgeCOM
 Running with large numbers of ranks and nodes
 ---------------------------------------------
 
+Running MirgeCOM on large systems can be challenging due to the startup overheads of
+Python and the MirgeCOM-related packages, as well as due to caching effects of kernels.
+As a general rule, make sure to execute MirgeCOM on a parallel file system, not on
+NFS-based file systems. On Quartz and Lassen, for example, this would mean running on the
+`/p/lscratchh/` and `/p/gpfs1/` file systems, respectively.
```

Comment:
Link to relevant documentation here (if possible).

Reply:
Good point, added.